src/EntryPoint/RestEntryPoints.ts

```diff
@@ -51,11 +51,39 @@ export class RestEntryPoints implements EntryPoints {
     });
   }
 
-  public async entrypoints(projectId: string): Promise<EntryPoint[]> {
-    const res = await this.client.get(
-      `/api/v2/projects/${projectId}/entry-points`
-    );
+  public async entrypoints(
+    projectId: string,
+    limit: number = 10
+  ): Promise<EntryPoint[]> {
+    let l = limit;
+    const batchSize = 50;
+    const data: EntryPoint[] = [];
+    let nextId: string;
+    let nextCreatedAt: string;
 
-    return res.data.items;
+    while (l > 0) {
+      const res = await this.client.get(
+        `/api/v2/projects/${projectId}/entry-points`,
+        {
+          params: {
+            limit: Math.min(l, batchSize),
+            nextId,
+            nextCreatedAt
```

Comment:
Keep the shorthand props first to improve readability

```suggestion
            nextId,
            nextCreatedAt,
            limit: Math.min(l, batchSize)
```